build-system/tasks/make-golden.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var argv = require('minimist')(process.argv.slice(2));
+//var exec = require('gulp-exec');
+var exec = require('child_process').exec;
+var gulp = require('gulp');
+var util = require('gulp-util');
+
+function makeGolden() {
+  var path = argv.path;
+  exec('phantomjs --ssl-protocol=any --ignore-ssl-errors=true \\' +
+       '--load-images=true \\' +
+      'testing/screenshots/make-screenshot.js http://localhost:8000 \\' +
+      '/examples.build/everything.amp.max.html \\' +
```

Comment:
we can pass it as option using minimist `var argv = require('minimist')(process.argv.slice(2));` at the top of the file.

`gulp make-golden path=blah opt2=blah`

unfortunately we can't do `gulp make-golden <PATH>` since gulp will think that `<PATH>` is another task it can execute.